doctr/datasets/generator/base.py

```diff
@@ -105,9 +105,9 @@ class _WordGenerator(AbstractDataset):
     def __init__(
         self,
         vocab: str,
-        min_chars: int,
-        max_chars: int,
         num_samples: int,
+        min_chars: int = 1,
+        max_chars: int = 17,  # good match for 32x128 images with different fonts
```

Comment:
The fact that our training script are used for specific shapes should not impact defaults on data generation pipelines I'd argue. I'm not sure we really need the default values here (making the text recognition script use this word generator with default values would be a better option in my opinion)